app/src/main/res/layout-land/admin_pin_activity.xml

```diff
@@ -57,36 +56,56 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/admin_pin_main_text" />
 
-      <org.oppia.android.app.profile.ProfileInputView
-        android:id="@+id/input_pin"
+      <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/admin_pin_input_pin"
+        style="@style/TextInputLayout"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginStart="168dp"
         android:layout_marginTop="48dp"
         android:layout_marginEnd="168dp"
-        app:inputLength="5"
-        app:isPasswordInput="true"
-        app:label="@string/admin_pin_new_pin"
+        app:errorMessage="@{viewModel.pinErrorMsg}"
```

Comment:
reformat and rearrange, same for other xml file

Reply:
Done